src/controllers/main/main.ts

```diff
@@ -902,6 +905,7 @@ export class MainController extends EventEmitter {
 
     const isUpdatingAccount = this.accounts.statuses.updateAccountState !== 'INITIAL'
 
+    this.selectedAccount.portfolio = DEFAULT_SELECTED_ACCOUNT_PORTFOLIO
```

Comment:
It'd be much better if the controller had a method called `resetSelectedAccountPortfolio`